crates/router/src/connector/datatrans.rs

```diff
@@ -193,11 +195,15 @@ impl ConnectorIntegration<api::Authorize, types::PaymentsAuthorizeData, types::P
 
     fn get_url(
         &self,
-        _req: &types::PaymentsAuthorizeRouterData,
+        req: &types::PaymentsAuthorizeRouterData,
         connectors: &settings::Connectors,
     ) -> CustomResult<String, errors::ConnectorError> {
         let base_url = self.base_url(connectors);
-        Ok(format!("{base_url}v1/transactions/authorize"))
+        if req.is_three_ds() {
```

Comment:
Please test PSYNC, Capture, Refunds and Incremental Auth for 3DS transactions.

Reply:
tested ,
For incremental Auth we have a different merchant account. Not able to test 3ds + incremental Auth
Except that everything is working